Gems/OpenParticleSystem/gem.json

```diff
@@ -0,0 +1,19 @@
+{
+    "gem_name": "OpenParticleSystem",
+    "display_name": "OpenParticleSystem",
+    "tags": ["OpenParticleSystem"],
```

Comment:
"tags" isn't a thing as far as I know. We always add the name of the object to the user_tags